engine/classes/Elgg/Di/PublicContainer.php

```diff
@@ -2,58 +2,29 @@
-
 /**
  * Public service container
  *
- * @property-read Accounts              $accounts        User accounts service
- * @property-read Config                $config          Config
- * @property-read Csrf                  $csrf            CSRF protection
- * @property-read Database              $db              Public database
- * @property-read EventsService         $events          Event service
- * @property-read FieldsService         $fields          Fields service
- * @property-read Gatekeeper            $gatekeeper      Gatekeeper
- * @property-read Tools                 $group_tools     Group Tools
- * @property-read HtmlFormatter         $html_formatter  HTML formatter
- * @property-read PluginHooksService    $hooks           Hooks service
- * @property-read LocaleService         $locale          LocaleService
- * @property-read Logger                $logger          Logger
- * @property-read Service               $menus           Menus
- * @property-read MimeTypeService       $mimetype        MIME type detection
- * @property-read ElggSession           $session         Session
- * @property-read SystemMessagesService $system_messages System messages
- * @property-read ColumnFactory         $table_columns   Table columns
- * @property-read Translator            $translator      Translator
- *
- * @method string echo (string $message_key, array $args = [], string $language = null) Outputs a translated string
+ * @property-read Elgg\Users\Accounts                  $accounts        User accounts service
+ * @property-read Elgg\Config                          $config          Config
+ * @property-read Elgg\Security\Csrf                   $csrf            CSRF protection
+ * @property-read Elgg\Application\Database            $db              Public database
+ * @property-read Elgg\EventsService                   $events          Event service
+ * @property-read Elgg\Forms\FieldsService             $fields          Fields service
+ * @property-read Elgg\Gatekeeper                      $gatekeeper      Gatekeeper
+ * @property-read Elgg\Groups\Tools                    $group_tools     Group Tools
+ * @property-read Elgg\Views\HtmlFormatter             $html_formatter  HTML formatter
+ * @property-read Elgg\PluginHooksService              $hooks           Hooks service
+ * @property-read Elgg\I18n\LocaleService              $locale          LocaleService
+ * @property-read Elgg\Logger                          $logger          Logger
+ * @property-read Elgg\Menu\Service                    $menus           Menus
+ * @property-read Elgg\Filesystem\MimeTypeService      $mimetype        MIME type detection
+ * @property-read ElggSession                          $session         Session
+ * @property-read Elgg\SystemMessagesService           $system_messages System messages
+ * @property-read Elgg\Views\TableColumn\ColumnFactory $table_columns   Table columns
+ * @property-read Elgg\I18n\Translator                 $translator      Translator
```

Comment:
In the InternalContainer all classes start with an `\`

Reply:
changed